src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Better PWA icons with maskable icons
+subhead: A new icon format to use adaptive icons on supporting platforms.
+description: |
+  A description of the article that will appear in search results.
+authors:
+  - tigeroakes
+date: 2019-12-09
+tags:
+  - post
+  - capabilities
+  - progressive-web-apps
+---
+
+## What are maskable icons? {: #what }
+
+<figure class="w-figure w-figure--inline-right">
+  <video class="w-screenshot" autoplay loop muted playsinline aria-label="Different masks applied to a maskable icon">
+    <source src="fugu-mask.webm" type="video/webm; codecs=vp8">
+    <source src="fugu-mask.mp4" type="video/mp4; codecs=h264">
+  </video>
+</figure>
```

Comment:
I like this animation, but to make it extra clear what maskable icons are about we could show some home screen screenshots. Kind of like how @petele did it in his video here: https://www.youtube.com/watch?v=L0OB0_bO5I0&t=24s @petele Do you still have these image resources and could we use them here?

Reply:
I added my own variant of those resources as the hero. Do you think I should remove the animation or keep both the hero and video?

![Icons contained inside white circles compared to icons covering its entire circle](https://user-images.githubusercontent.com/1782266/70840400-d905fd80-1dc6-11ea-83be-fad1c8f769af.png)

Reply:
That looks great!

The hero is not showing up for me on [the preview URL](https://deploy-preview-1975--web-dev-staging.netlify.com/maskable-icon/).

Reply:
It's showing now.

Reply:
**Update:** Never mind, it does now. Caching is hard? :D

Reply:
IMHO we should repeat the relevant parts of the hero image in the body, in addition to your animation. Something like this:

```
Without explicit maskable icons, your PWA icons might look like this:

insert top part of <https://user-images.githubusercontent.com/81942/70888356-70926880-1fe0-11ea-8a12-75cf41e3051a.png>

With maskable icons, you have more control etc. etc.

insert bottom part of <https://user-images.githubusercontent.com/81942/70888356-70926880-1fe0-11ea-8a12-75cf41e3051a.png>

Note that the shape is platform-specific: it could be a circle, a squircle, etc. etc.

insert your animation
```

Reply:
Updated!